### ISSUE TITLE
Bugged DK Quest

### ISSUE THREAD

**Issue by bdhamp28** [dalaranwow/dalaran-wow#4877]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
 I was doing a Death Knight chain quest (the one from Scarlet Tavern where i had to go to the towers and bring back someones head-I believe the reward was the blue chest piece) and I died while doing the quest. When I resurrected, the room was empty except for literally 13 Tauren DKs all sitting in the corner (which was very strange).  I ran back to quest giver in Scarlet Tavern and abandoned the quest, but was not able to get it back.  This is the only quest that was available to me and leads to other quests that eventually lead me out of the starting zone.  So as it sits right now, I am dead in the water with no available quests and nowhere to go.

**Expected behaviour**: Tell us what should happen instead
Since the quest bugged out, I was hoping that I could abandon it and get it again. 
**Steps to reproduce the problem**:

1. I tried logging off and back on multiple times.
2. I tried exiting the game and signing back in multiple times.
3. I tried running back and forth from the castle to the tavern in attempt to see if the NPC would return.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. (Or create a new Ingame-Account in order to be able to talk with a gamemaster with you new level 1 character) Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
